pcl_ros/src/pcl_ros/filters/filter.cpp

```diff
@@ -247,20 +247,11 @@ pcl_ros::Filter::add_common_params()
     "Set to true if we want to return the data outside [filter_limit_min; filter_limit_max].";
   declare_parameter(flneg_desc.name, rclcpp::ParameterValue(false), flneg_desc);
 
-  rcl_interfaces::msg::ParameterDescriptor keep_organized_desc;
```

Comment:
We don't need this in `VoxelGrid` so let's move it to the constructor for `Passthrough`.

Reply:
I agree with this